### PR TITLE
remodel: init at 0.10.0

### PR DESCRIPTION
**Pull request by wackbyte**
###### Description of changes

[Remodel](https://github.com/rojo-rbx/remodel) is a Roblox file manipulation tool.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ajs124** [comment]
This does not seem to build on darwin, so you might want to restrict `meta.platforms`

**Comment by OPNA2608** [comment]
@ajs124 `meta.platforms` is the wrong place for this. The package *should* support Darwin, but the build is *broken*. `meta.broken` is the correct place for such a marking.

Bildschirmfoto von 2022-07-31 20-59-10

File: pkgs/development/tools/remodel/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  cargoSha256 = "sha256-b9+eV2co4hcKLZxJRqDIX2U0O25Ba5UHQiNpfjE4fN4=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
```
Comment (OPNA2608) [inline pkgs/development/tools/remodel/default.nix:28]:
```suggestion
  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
    Security
  ];
```

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -20768,6 +20768,8 @@ with pkgs;
 
   randomx = callPackage ../development/libraries/randomx { };
 
+  remodel = callPackage ../development/tools/remodel { };
```
Comment (OPNA2608) [inline pkgs/top-level/all-packages.nix:20771]:
```suggestion
  remodel = callPackage ../development/tools/remodel {
    inherit (darwin.apple_sdk.frameworks) Security;
  };
```

File: pkgs/development/tools/remodel/default.nix [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+}:
```
Comment (OPNA2608) [inline pkgs/development/tools/remodel/default.nix:7]:
```suggestion
, Security
}:
```

**Review by OPNA2608 — CHANGES_REQUESTED** [review]
Fixes for Darwin.

Screenshot_macOS_2022-07-31_20:57:40

**Comment by IvarWithoutBones** [comment]
This should've gone in `pkgs/tools/games` instead.

**Pull request merged**